src/ncp/ncp_base.hpp

```diff
@@ -293,6 +293,7 @@ class NcpBase
 
 #if OPENTHREAD_ENABLE_NCP_VENDOR_HOOK
     otError VendorCommandHandler(uint8_t aHeader, unsigned int aCommand);
+    void VendorHandleFrameRemovedFromNcpBuffer(NcpFrameBuffer::FrameTag aFrameTag);
```

Comment:
The method name is rather long, but we decided to keep it the same as existing one in `NcpBase`.